packages/apollo-server-hapi/src/__tests__/ApolloServer.test.ts

```diff
@@ -75,28 +75,16 @@ describe('apollo-server-hapi', () => {
 
   describe('constructor', () => {
     it('accepts typeDefs and resolvers', async () => {
-      const app = new Server();
-      const server = new ApolloServer({ typeDefs, resolvers });
-      await server.start();
-      return server.applyMiddleware({ app });
+      return createServer({ typeDefs, resolvers });
```

Comment:
I'd find `await` rather than `return` here more clear (I recognize the original code used `return`).